starboard/raspi/shared/launcher.py

```diff
@@ -260,28 +260,30 @@ def _PexpectSendLine(self, cmd):
 
   def _PexpectReadLines(self):
     """Reads all lines from the pexpect process."""
-    # pylint: disable=unnecessary-lambda
-    @retry.retry(
-        exceptions=Launcher._RETRY_EXCEPTIONS,
-        retries=Launcher._PEXPECT_READLINE_TIMEOUT_MAX_RETRIES,
-        backoff=lambda: self.shutdown_initiated.is_set(),
-        wrap_exceptions=False)
-    def _readloop():
-      while True:
-        # Sanitize the line to remove ansi color codes.
-        line = Launcher._PEXPECT_SANITIZE_LINE_RE.sub(
-            '', self.pexpect_process.readline())
-        self.output_file.flush()
-        if not line:
-          return
-        # Check for the test complete tag. It will be followed by either a
-        # success or failure tag.
-        if line.startswith(self.test_complete_tag):
-          if line.find(self.test_success_tag) != -1:
-            self.return_value = 0
-          return
-
-    _readloop()
+
+    def _readline():
+      # Sanitize the line to remove ansi color codes.
+      return Launcher._PEXPECT_SANITIZE_LINE_RE.sub(
+          '', self.pexpect_process.readline())
```

Comment:
I think just the pexpect_process.readline should be in the retry wrapper

Reply:
Done